src/SFA.DAS.Authorization/AuthorizationContext.cs

```diff
@@ -30,5 +31,10 @@ public bool TryGet<T>(string key, out T value)
 
             return exists;
         }
+
+        public string GetDescription()
```

Comment:
Would there be any benefit in overriding ToString() for this?

Reply:
Agreed, fixed.